src/rfac/crfrs.c

```
@@ -0,0 +1,167 @@
+#define RS_BETA 0.15
+
+struct cr_rs_arrays {
+  real *e_d1;
+  real *e_d2;
+  real *t_d1;
+  real *t_d2;
+  real *e_y;
+  real *t_y;
+};
+
+static real cr_rs_y(real intensity, real deriv, real deriv2, real vi)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'cr_rs_y' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)